frontend/src/pages/pipelines/global/modelCustomization/ModelCustomizationForm.tsx

```diff
@@ -10,19 +11,33 @@ import GenericSidebar from '~/components/GenericSidebar';
 import EnsureAPIAvailability from '~/concepts/pipelines/EnsureAPIAvailability';
 import { useValidation, ValidationContext } from '~/utilities/useValidation';
 import useGenericObjectState from '~/utilities/useGenericObjectState';
-import { modelCustomizationFormSchema } from '~/concepts/pipelines/content/modelCustomizationForm/modelCustomizationFormSchema/validationUtils';
+import {
+  ModelCustomizationFormData,
+  modelCustomizationFormSchema,
+} from '~/concepts/pipelines/content/modelCustomizationForm/modelCustomizationFormSchema/validationUtils';
 import { usePipelinesAPI } from '~/concepts/pipelines/context';
 import { modelCustomizationRootPath } from '~/routes';
 import { useIlabPipeline } from '~/concepts/pipelines/content/modelCustomizationForm/useIlabPipeline';
 import FineTunePage from './FineTunePage';
 import { FineTunePageSections, fineTunePageSectionTitles } from './const';
 
+const BASE_MODEL_INPUT_STORAGE_LOCATION_URI_PARAM_NAME = 'baseModelInputStorageLocationUri';
```

Comment:
Can we shorten the variable name to `BASE_MODEL_LOCATION_URI`

Reply:
I can do it but I'd rather keep a more informative name. `BASE_MODEL_LOCATION_URI` doesn't say to me that it refers to Input Storage.

Reply:
then just remove the PARAM_NAME?

Reply:
How about `BASE_MODEL_INPUT_STORAGE_LOCATION_URI_KEY`?

I'd expect to see a URI as the value of a const named like `BASE_MODEL_INPUT_STORAGE_LOCATION_URI`